components/CheckInOutForm/index.tsx

```diff
@@ -107,6 +119,22 @@ function CheckInOutForm({
     ...initialFormData,
   })
 
+  const updateTextFieldAttributes = (
+    _e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
```

Comment:
Change this to accept a string or a number. See other comment on the TextField components on line 272 and 284 for more details